app/styles/components/_notifications.scss

```diff
@@ -0,0 +1,153 @@
+$notifications-dropdown-content-color: $wds-color-light-c7;
```

Comment:
can we move those variables into styles block, they are not exposed publicly then.